### PR TITLE
Added Cisco IOS "show alert counters"

### PR DESCRIPTION
**Pull request by mjuenema**
##### ISSUE TYPE
 - New Template Pull Request

##### COMPONENT
- ```cisco_ios_show_alert_counters.textfsm``` template including example input and output.

##### SUMMARY
Added ```cisco_ios_show_alert_counters``` template to parse output of ```show alert counters``` command which is available 
on the IE-3000 Industrial Ethernet switches running IOS 15.x and possibly others.

### CODE REVIEW

**Comment by mjuenema** [comment]
For some obscure reason the tox test fails when the ```black``` command wants to re-format files which I haven't even changed. Even the ```master``` branch fails on my clone.
There seems to be a difference between black releases 19.x and 20.x. I opened a separate issues #794 for this.

File: tests/cisco_ios/show_alert_counters/cisco_ios_show_alert_counters.raw [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+Interface: Fa1/5
+
+Interface: Fa1/6
+
+Interface: Fa1/7
+
+Interface: Fa1/8
+
+Interface: Gi1/1
+
```
Comment (FragmentedPacket) [inline tests/cisco_ios/show_alert_counters/cisco_ios_show_alert_counters.raw:51]: This template should still capture all the interfaces.

Look into the `-> Continue.Record` option to capture the same output you have plus the additional blank interfaces.

**Review by FragmentedPacket — CHANGES_REQUESTED** [review]

File: templates/cisco_ios_show_alert_counters.textfsm [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+Value description (.+)
+Value recommendation (.+)
+
+Start
+  ^Interface:\s+${interface} -> Clear
+  ^Error Code:\s+${errorcode}
+  ^Timestamp:\s+${timestamp}
+  ^Description:\s+${description}
+  ^Recommendation:\s+${recommendation} -> Record
+
```
Comment (jmcgill298) [inline templates/cisco_ios_show_alert_counters.textfsm:13]: Need to match all possible lines, and add an error for any line that is not matched

**Review by jmcgill298 — CHANGES_REQUESTED** [review]

**Comment by FragmentedPacket** [comment]
Closing in favor of #881. @mjuenema Thanks for your contribution! I added the suggestions from @jmcgill298 to the new PR, and your commit is on that PR for credit for the work.

**Pull request closed without merging**